tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/mqtt/imported/MQTTTest.java

```diff
@@ -1001,91 +1005,93 @@ public void testCleanSession() throws Exception {
-   //      activeMQConnection.setUseRetroactiveConsumer(true);
-   //      activeMQConnection.start();
-   //      Session s = activeMQConnection.createSession(false, Session.AUTO_ACKNOWLEDGE);
-   //      javax.jms.Topic jmsTopic = s.createTopic(destinationName);
-   //      MessageProducer producer = s.createProducer(jmsTopic);
-   //
-   //      // send retained message from JMS
-   //      final String RETAINED = "RETAINED";
-   //      TextMessage sendMessage = s.createTextMessage(RETAINED);
-   //      // mark the message to be retained
-   //      sendMessage.setBooleanProperty(RetainedMessageSubscriptionRecoveryPolicy.RETAIN_PROPERTY, true);
-   //      // MQTT QoS can be set using MQTTProtocolConverter.QOS_PROPERTY_NAME property
-   //      sendMessage.setIntProperty(MQTTProtocolConverter.QOS_PROPERTY_NAME, 0);
-   //      producer.send(sendMessage);
-   //
-   //      provider.subscribe("foo/+", AT_MOST_ONCE);
-   //      byte[] message = provider.receive(10000);
-   //      assertNotNull("Should get retained message", message);
-   //      assertEquals(RETAINED, new String(message));
-   //
-   //      for (int i = 0; i < NUM_MESSAGES; i++) {
-   //         String payload = "This is Test Message: " + i;
-   //         sendMessage = s.createTextMessage(payload);
-   //         producer.send(sendMessage);
-   //         message = provider.receive(5000);
-   //         assertNotNull("Should get a message", message);
-   //
-   //         assertEquals(payload, new String(message));
-   //      }
-   //      provider.disconnect();
-   //      activeMQConnection.close();
-   //   }
+      final byte[] payload = RETAINED.getBytes();
+
+      Connection connection = cf.createConnection();
+      // MUST set to true to receive retained messages
+      connection.start();
+
+      Session s = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+      javax.jms.Queue jmsQueue = s.createQueue(jmsTopicAddress);
+      MessageConsumer consumer = s.createConsumer(jmsQueue);
+
+      provider.publish(address, RETAINED.getBytes(), AT_LEAST_ONCE, true);
+
+      // check whether we received retained message on JMS subscribe
+      BytesMessage message = (BytesMessage) consumer.receive(5000);
+      assertNotNull("Should get retained message", message);
+
+      byte[] b = new byte[8];
+      message.readBytes(b);
+      assertArrayEquals(payload, b);
+
+      for (int i = 0; i < NUM_MESSAGES; i++) {
+         String p = "Test Message: " + i;
+         provider.publish(address, p.getBytes(), AT_LEAST_ONCE);
+         message = (BytesMessage) consumer.receive(5000);
+         assertNotNull("Should get a message", message);
+
+         byte[] bytePayload = new byte[p.getBytes().length];
+         message.readBytes(bytePayload);
+         assertArrayEquals(payload, b);
+      }
+
+      connection.close();
+      provider.disconnect();
+   }
+
+//   @Test(timeout = 2 * 60 * 1000)
+//   public void testSendJMSReceiveMQTT() throws Exception {
```

Comment:
shouldn't you just remove the test?

or keep it in with an ignored annotation if it doesn't pass?